curp/src/sync_manager.rs

```diff
@@ -203,22 +221,18 @@ impl<C: Command + 'static> SyncManager<C> {
         mut ae_trigger_rx: UnboundedReceiver<usize>,
     ) {
         while let Some(i) = ae_trigger_rx.recv().await {
-            if state.read().is_leader() {
-                // send append_entries to each server in parallel
-                let rpcs: FuturesUnordered<_> = connects
-                    .iter()
-                    .map(|connect| {
-                        Self::send_log_until_succeed(
-                            i,
-                            Arc::clone(connect),
-                            Arc::clone(&state),
-                            commit_trigger.clone(),
-                        )
-                    })
-                    .collect();
-                let _drop = tokio::spawn(async move {
-                    let _drop: Vec<_> = rpcs.collect().await;
-                });
+            if !state.read().is_leader() {
+                continue;
```

Comment:
It should not happen, please log something for the later diagnose.

Reply:
Added.